README.md

```diff
@@ -79,7 +79,28 @@ docker run -d --name radicale \
     tomsquest/docker-radicale
 ```
   
-Note: On the Linux Kernel capabilities, `CHOWN`, `SETUID` and `SETGID` are for fixing the permission of the mounted volume. `KILL` is to allow Radicale to exit.
+Note on capabilities:
+- `CHOWN` is used to restore the permission of the `data` directory. Skip it if not chowning (see below)
+- `SETUID` and `SETGID` are used to changed the `UID` and `GID` of the user if asked. Skip them if not changing UID/GID (see below)
```

Comment:
SETUID and SETGID are required for su-exec to work, so are always needed.

Reply:
Totally right ! :+1:
Fixed.